### ISSUE TITLE
Option max_lines_per_load has no default

### ISSUE THREAD

**Issue by adborden** [caciviclab/disclosure-backend#28]
```
$ python disclosure-backend/manage.py downloadnetfilerawdata
Traceback (most recent call last):
  File "disclosure-backend/manage.py", line 9, in <module>
    execute_from_command_line(sys.argv)
  File "/home/adborden/.virtualenvs/disclosure-backend/lib/python2.7/site-packages/django/core/management/__init__.py", line 338, in execute_from_command_line
    utility.execute()
  File "/home/adborden/.virtualenvs/disclosure-backend/lib/python2.7/site-packages/django/core/management/__init__.py", line 330, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/adborden/.virtualenvs/disclosure-backend/lib/python2.7/site-packages/django/core/management/base.py", line 393, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/home/adborden/.virtualenvs/disclosure-backend/lib/python2.7/site-packages/django/core/management/base.py", line 444, in execute
    output = self.handle(*args, **options)
  File "netfile/management/commands/downloadnetfilerawdata.py", line 90, in handle
    self.max_lines_per_load = int(options.get('max_lines_per_load'))
TypeError: int() argument must be a string or a number, not 'NoneType'
```

**Comment by adborden** [comment]
I've worked around the issue, but I'm not really sure where these options came from. They don't seem to be referenced in django-calaccess-raw-data, and I don't see them specified in any of our cron stuff. I suppose the two workarounds are the right fix in this case.

**Comment by bcipolli** [comment]
Fixed in f98d3ab
